CMSgov/onemac
# OY2 21206: Change architecture to make Waiver Children into a Flat Dashboard

Story: https://qmacbis.atlassian.net/browse/OY2-21206
Endpoint: See github-actions bot comment

### Details
Flatten the Waiver Dashboard so there are no expanding items... Amendments and Waiver Extensions get moved to the top level.

### Changes
- renamed newSubmission to newComponent so that we can reuse with incoming SEATool Components
- Removed submitterId, children, and submissionId from stored items
- Added currentStatusTimestamp to track when the package status was last changed (also in anticipation of SEATool updates), normalized to match the submissionTimestamp for OneMAC submissions.
- Updated GSI1pk and GSI1sk to include Waiver Amendments and Temporary Extensions
- Created GSI2 index to hold the parent information previously overloaded into GSI1
- removed extraneous fields from the MyPackages data projections
- updated migrate to move Amendments and Extensions to top level
- updated Waiver Family parsing to allow '-' s
- Removed "expandable" code, icons, tests, column, etc from Dashboard table

### Implementation Notes
- I thought some of the SEATool integration pieces would be more applicable to this story's implementation... they were not, but I am reluctant to remove them now that they are done!

### Test Plan
1. Log in as any (and all) Users
2. Navigate to the Package Dashboard, Waivers Tab
3. Verify that Amendments and Temporary Extensions are at the top level
4. Verify there are no children showing and no expanders
